WooCommerce/Classes/Analytics/WooAnalytics.swift

```diff
@@ -227,7 +227,9 @@ private extension WooAnalytics {
         let widgetAnalyticNames: [String] = installedWidgets.map { widgetInfo in
             switch widgetInfo.kind {
             case WooConstants.storeInfoWidgetKind:
-                return WooAnalyticsEvent.Widgets.Name.todayStats.rawValue
+                return "\(WooAnalyticsEvent.Widgets.Name.todayStats.rawValue)-\(widgetInfo.family)"
```

Comment:
How can we make sure that these match Android events, so we can continue to write bundled reports?

Reply:
`today-stats` became `today-stats-systemMedium`. We can make exception to it and log without a suffix to keep alignment with Android, what do you think?

Lockscreen will still have stuff like `today-stats-accessoryCircular`.